### PR TITLE
automember: Fix action to be automember or member, not service

### PR DESCRIPTION
**Pull request by t-woerner**
The action is service and member for the module, but it needs to be the
module name and member.

### CODE REVIEW

**Review by rjeffman — APPROVED** [review]
lgtm.

File: plugins/modules/ipaautomember.py [hunk 1]
```diff
@@ -72,9 +72,9 @@
     elements: dict
     aliases: ["automemberinclusiveregex"]
   action:
-    description: Work on service or member level
-    default: service
-    choices: ["member", "service"]
+    description: Work on automember or member level
+    default: automember
+    choices: ["member", "automember"]
```
Comment (varunmylaraiah) [inline plugins/modules/ipaautomember.py:77]: I think automemberrule is preferable instead of automember.

Comment (t-woerner) [reply plugins/modules/ipaautomember.py:77]: The default action is always the module name without the "ipa" prefix.

I agree that the name automemberrule would be a lot better, but the IPA API is also using "automember" without "rule".

Comment (varunmylaraiah) [reply plugins/modules/ipaautomember.py:77]: Sure, No issues

**Pull request merged**